testapps/test_pillowtop/tests/test_case_pillow.py

```diff
@@ -49,6 +53,24 @@ def test_case_pillow(self):
         self.assertEqual(case_id, case_doc['_id'])
         self.assertEqual(case_name, case_doc['name'])
 
+    @run_with_all_backends
+    def test_case_pillow_error_in_case_es(self):
+        pillow_errors = PillowError.objects.filter(pillow='case-pillow').count()
```

Comment:
I don't know why, but my local tests always started with 1 pillow error. I think that may just be dirty-ness in my test setup and this isn't required, but I added here to get my local tests to pass. would be appreciated if someone else could test that

Reply:
I also have one error locally.

Reply:
I think it's because it's running twice (once for each backend) so the second run has an error already in the DB.

Reply:
https://github.com/dimagi/commcare-hq/pull/25482/commits/14b8bcf5d14597fd212e8acba80cf8becc8c255e